Makefile

```
@@ -179,7 +178,7 @@ endef
 endif
 
 INDEXED_YAML := $(YQ) e '{([.metadata.name, .kind] | join("-")): .}'
-helm-test-operator: manifests helm-cmd kustomize yq dyff
+helm-test-operator: manifests helm-cmd kustomize yq dyff ## test operator helm chart against kustomize
```

```suggestion
helm-test-operator: manifests helm-cmd kustomize yq dyff ## Test operator helm chart against kustomize
```

consistent casing with elsewhere

---

fixed in [16138cb](https://github.com/sky-uk/kfp-operator/pull/486/commits/16138cb7a8661c98290d34dceb787c04048ec498)